core/src/main/scala/org/apache/spark/storage/BlockManagerMessages.scala

```diff
@@ -132,6 +132,8 @@ private[spark] object BlockManagerMessages {
   case class GetReplicateInfoForRDDBlocks(blockManagerId: BlockManagerId)
     extends ToBlockManagerMaster
 
+  case object GetAllExecutors extends ToBlockManagerMaster
```

Comment:
I don't think we still need this if we are tracking historical executor locations for merger locations.